docs/docs/native-docker.md

```diff
@@ -280,12 +279,11 @@ the future, as Mesos may not always interact with Docker via the CLI.
 }
 ```
 
-#### Docker container support without Docker Engine
+#### Docker Container Support without Docker Engine
```

Comment:
Let's change this to "# Configure Your application to use the Mesos containerizer". Right now this is a subheader in the Docker containerizer topic, but it shouldn't.